media.css

```diff
@@ -0,0 +1,442 @@
+@media screen and (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .menu2 {
+    display: flex;
+    flex-direction: column;
+    justify-content: right;
+    padding-right: 154px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 20px;
+  }
+
+  .menu2 > ul {
+    display: flex;
+    list-style: none;
+    gap: 32px;
+  }
+
+  .Main-container {
+    margin-left: 60px;
+  }
+
+  .top {
+    display: flex;
+    flex-direction: column;
+    height: 750px;
+    background-image: url(assets/images/Header\ bg.svg);
+    background-color: white;
+    background-repeat: no-repeat;
+  }
+
+  h1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: bold;
+    font-size: 48px;
+    line-height: 60px;
+    margin-left: 145px;
+    max-width: 30%;
+    width: 30%;
+    margin-top: 185px;
+    color: #172b4d;
+  }
+
+  .About {
+    font-family: 'Poppins', sans-serif;
+    font-style: regular;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin-top: -12px;
+    margin-left: 150px;
+    margin-right: 350px;
+    margin-bottom: 15px;
+  }
+
+  .Meet {
+    margin-left: 150px;
+  }
+
+  .Links ul {
+    margin-left: 112px;
+  }
+
+  .img1 {
+    display: none;
+  }
+
+  .works {
+    background: none;
+    display: grid;
+    grid-template-columns: auto;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .card1 {
+    display: flex;
+    padding: 24px;
+  }
+
+  .card2 {
+    display: flex;
+    padding: 24px;
+    width: 90%;
+  }
+
+  .project1 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
+    margin: auto;
+    margin-top: 110px;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  .project2 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row-reverse;
+    text-align: start;
+    box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.75);
+    box-decoration-break: unset;
+    width: 1156px;
+    height: 496px;
```

Comment:
I would suggest using of percentages in place of pixels so that it doesn't break on small screens.